.pytool/Plugin/LineEndingCheck/LineEndingCheck.py

```diff
@@ -84,6 +84,30 @@ def GetTestName(self, packagename: str, environment: VarDict) -> Tuple:
         return ("Check line endings in " + packagename, packagename +
                 "." + PLUGIN_NAME)
 
+    # Note: This function access git via the command line
+    #
+    #   function to check and warn if git config reports that 
+    #   autocrlf is configured to TRUE
+    def _check_autocrlf(self):
+
+        outstream_buffer = StringIO()
+        exit_code = RunCmd("git", "config --get core.autocrlf",
+                           outstream=outstream_buffer);
+        if exit_code == 0:
+            result = outstream_buffer.getvalue().strip()
+            if result.upper() != "FALSE":
+                logging.warning(f"git config core.autocrlf is set to {result}, recommended setting is false")

```

Comment:
Keep alignment as close to 80 as possible.

```suggestion
                logging.warning(f"git config core.autocrlf is set to {result}"
                                f", recommended setting is false")
```